cli/crates/cli/tests/utils/environment.rs

```diff
@@ -83,12 +91,22 @@ impl Environment {
 
         drop(file);
     }
-}
 
-impl Drop for Environment {
-    fn drop(&mut self) {
+    pub fn kill_processes(&mut self) {
         self.commands.iter().for_each(|command| {
             kill_with_children(*command.pids().first().unwrap());
         });
+
+        self.commands = vec![];
+    }
+
+    pub fn has_dot_grafbase_folder(&mut self) -> bool {
```

Comment:
Hm, I would use the `directory` word consistently.

Reply:
Done, had to make changes in other places as well so please take another look. Thanks!